<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infodisplay</class>
 <widget class="QWidget" name="infodisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="stabilisLegal" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>28</y>
     <width>704</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="qtLegal" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>206</y>
     <width>704</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="iconsLegal" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>384</y>
     <width>704</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
